source/wp-content/themes/wporg-developer-2023/inc/shortcodes.php

```diff
@@ -57,6 +57,17 @@ function() {
 	}
 );
 
+/**
+ * Get the title of the article.
+ */
+add_shortcode(
+	'article_title',
+	function() {
+		global $post;
+		return $post->post_title;
```

Comment:
Maybe `get_the_title()` so that the title filters can be run? Not a strong suggestion.

Reply:
I think I tried that but it didn't work, I'll take another look